doc/release/1.14.0-notes.rst

```diff
@@ -93,6 +93,13 @@ This change affects only ``float32`` and ``float16`` arrays.
 This is for pytest compatibility in the case of duplicate test file names in
 the different directories. As a result, ``run_module_suite`` no longer works,
 i.e., ``python <path-to-test-file>`` results in an error.
+``MaskedArray.squeeze`` never returns ``np.ma.masked``
```

Comment:
@eric-wieser It was mentioned that the file would need editing to fix the line spacing before release to fix this sort of problem, but that is easy to do. I just wish it worked better in general.